src/components/DataTable/DataTableEmpty.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import classnames from "classnames"
+
+interface DataTableEmptyProps {
+  className?: string
+}
+
+const DataTableEmpty: React.FC<DataTableEmptyProps> = ({ className }) => (
+  <div className={classnames("data-table-empty", className)}>
+    Data Table Empty
```

Comment:
Will need a way to set a message, eg. "No users matching criteria exist."

In situations when list came up empty due to search criteria we could use a button "Clear filters". We will keep filters in address so people can bookmark those and send them to other admins, so button could be displayed when `cleanFiltersUrl` prop is set.

Also this component will have dual use: in tables (eg. users list) and div based lists (eg. categories list). So maybe if `colspan` prop is set, this will be a tablerow with single cell with class `data-table-empty`?